source/Ship.h

```diff
@@ -596,6 +596,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Links between escorts and parents.
 	std::vector<std::weak_ptr<Ship>> escorts;
 	std::weak_ptr<Ship> parent;
+
+	std::map<std::string, int> *partsToRemove;
```

Comment:
Replace this with a bool: `ignoreBaseBays`. It is already possible to redefine the bays in a variant definition, as long as some bays are defined, the base bays will be ignored. In cases where no bays are desired, there is no way to have the game not import the bay layout of the base ship. This bool would tell the game not to do that in Ship::FinishLoading().
We should also consider expanding this to support a variant having no weapon hardpoints where the base ship did.

Reply:
looking for the long run, maybe a bitmap with offset type is better here?